compat/mingw/dbghelp.h

```diff
@@ -0,0 +1,249 @@
+// Copyright 2014 The Crashpad Authors. All rights reserved.
```

Comment:
Are these files upstream? I wonder why they are not part of our fork?

Reply:
Ah, this is a patch to the dbghelp.h header file of MinGW, which misses somes references required by crashpad. I just copy-pasted the whole file from /compat/non_win/dbghelp.h and removed the duplicated references :P